src/Aquifer.API/Endpoints/Notifications/List/Endpoint.cs

```diff
@@ -0,0 +1,197 @@
+                },
+                HelpDocument = null,
+                IsRead = readNotificationEntityIdsByNotificationKindMap.GetValueOrDefault(NotificationKind.Comment)
+                    ?.Contains(c.CommentId)
+                         ?? false,
+            })
+            .Concat(helpDocumentNotificationData
+                .Select(h => new Response
+                {
+                    Kind = NotificationKind.HelpDocument,
+                    Comment = null,
+                    HelpDocument = new HelpDocumentNotification
+                    {
+                        Id = h.Id,
+                        Created = h.Created,
+                        Title = h.Title,
+                        Type = h.Type,
+                        Url = h.Url,
+                        ThumbnailUrl = h.ThumbnailUrl,
+                    },
+                    IsRead = readNotificationEntityIdsByNotificationKindMap.GetValueOrDefault(NotificationKind.HelpDocument)
+                        ?.Contains(h.Id)
+                            ?? false,
+                }))
+            .Where(x => !req.IsRead.HasValue || x.IsRead == req.IsRead.Value)
+            .OrderByDescending(n => n.Comment?.Created ?? n.HelpDocument!.Created)
+            .Skip(req.Offset)
+            .Take(req.Limit)
+            .ToList();
+
+        await SendOkAsync(notifications, ct);
+    }
+
+    private sealed record HelpDocumentNotificationData(
+        int Id,
+        HelpDocumentType Type,
+        DateTime Created,
+        string Title,
+        string Url,
+        string? ThumbnailUrl);
+
+    private static async Task<IReadOnlyList<HelpDocumentNotificationData>> GetHelpDocumentNotificationDataAsync(
+        DbConnection dbConnection,
+         DateTime minimumHelpDocumentCreatedDate,
+         CancellationToken ct)
+    {
+        const string query = """
+            SELECT
+                d.Id,
+                d.Type,
+                d.Created,
+                d.Title,
+                d.Url,
+                d.ThumbnailUrl
+            FROM HelpDocuments d
+            WHERE
+                d.Created > @minimumHelpDocumentCreatedDate
+            ORDER BY
+                d.Created DESC,
+                d.Id DESC
+            """;
+
+        return (await dbConnection.QueryWithRetriesAsync<HelpDocumentNotificationData>(
+                query,
+                new { minimumHelpDocumentCreatedDate },
+                cancellationToken: ct))
+            .ToList();
+    }
+
+    private sealed record ResourceCommentNotificationData(
+        int CommentId,
+        string Comment,
+        DateTime Created,
+        int UserId,
+        string UserFirstName,
+        string UserLastName,
+        int ResourceContentId,
+        string ResourceEnglishLabel,
+        string ParentResourceDisplayName);
+
+    private static async Task<IReadOnlyList<ResourceCommentNotificationData>> GetResourceCommentNotificationDataAsync(
```

Comment:
The [previous logic ](https://github.com/BiblioNexusStudio/aquifer-server/blob/67b0cf92113516b88b4e1543945cffd178c5401d/src/Aquifer.Jobs/Subscribers/NotificationMessageSubscriber.cs#L100-L117) also limited notifications to only users in the same company as the user who commented.  That would add extra DB joins here that don't seem worth it so I left it as notifying anyone previously assigned to the resource content version.

I'd argue that it would also be useful to notify anyone who had previously commented on the thread as well but that wasn't in the previous logic so I didn't add it here.